website/docs/reference/resource-configs/alias.md

```diff
@@ -84,6 +86,35 @@ snapshots:
 This would return the name `analytics.finance.your_snapshot` in the database.
 
 </File>
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+Configure a snapshots's alias in your `dbt_project.yml` file or config block. 
+
+For example, if you have a snapshot that represents `your_snapshot` and want to alias it as `updated_at_id`, you would alias like this:
+
+<File name='dbt_project.yml'>
```

Comment:
I think this example shows setting this in a schema yml config file, not the dbt_project.yml

Reply:
Changed format to the project yaml